solutions/secure/main.tf

```diff
@@ -0,0 +1,29 @@
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.4"
+  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
+  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
+}
+
```

Comment:
To be consistent with other DAs, we probably want to add support to create a KMS key in an existing KMS instance. For example, see https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/30b001f78aac4b858d31a9e7dfec02db7be3220a/solutions/instances/main.tf#L25-L58